files/default/neutron-ha-tool.py

```diff
@@ -235,6 +239,7 @@ def run(args):
     qclient.format = 'json'
 
     configure(args, qclient)
+    Configuration.router_filter.load()
```

Comment:
Right, so first I wrote the tests that were asserting if all the parameters were injected to the instances based on the command line arguments. Now if we put the responsibility of opening the file into the constructor, those tests would fail without an existing file, making it "hard" to test the wiring section separately. That's the reason for separating the load into it's own function.